docs/src/modules/utils/generatePropTypeDescription.ts

```diff
@@ -47,20 +47,24 @@ function getChained(type: PropTypeDescriptor) {
   return false;
 }
 
-function escapeCell(value: string): string {
+export function escapeCell(value: string): string {
   // As the pipe is use for the table structure
   return value.replace(/</g, '&lt;').replace(/`&lt;/g, '`<').replace(/\|/g, '\\|');
 }
 
-function isElementTypeAcceptingRefProp(type: PropTypeDescriptor): boolean {
+function isIntegerPropType(type: PropTypeDescriptor): boolean {
+  return type.raw === 'integerPropType';
+}
+
```

Comment:
@oliviertassinari should we change `api-docs` jsons or should we wait after customPropType would be finished/tested ?

Reply:
I have reverted this change, it's off-topic for the problem we are going after (duplication).